etc/notebooks/got_scotch_demo/scotch_dashboard.ipynb

```diff
@@ -634,15 +735,41 @@
   {
    "cell_type": "markdown",
    "metadata": {
+    "urth": {
+     "dashboard": {
+      "hidden": true
+     }
+    }
+   },
+   "source": [
```

Comment:
Move the "Powered by data ..." cell above this section. Add markdown above it saying "Finally, we'll include a footer in our dashboard with some attribution information."